data/mods/Aftershock/mobs/uplifted_monsters.json

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "id": "mon_uplifted_bear",
+    "type": "MONSTER",
+    "name": { "str": "Schwarz Walder" },
+    "description": "Schwarz Walders were originally developed by a German company as forest rangers for the expanded Black Forest Conservation Area.  Shortly afterwards it was determined that they also made excellent long range reconnaissance units that could operate independently indefinitely.  Pre-Cataclysm they lived on every continent in a variety of jobs.",
+    "default_faction": "bear",
+    "bodytype": "bear",
+    "categories": [ "UPLIFT" ],
+    "species": [ "MAMMAL" ],
+    "volume": "90 L",
+    "weight": "120 kg",
+    "hp": 120,
+    "speed": 140,
+    "material": [ "flesh" ],
+    "symbol": "B",
+    "color": "dark_gray",
+    "aggression": -10,
+    "morale": 60,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 6,
+    "melee_cut": 4,
+    "dodge": 3,
+    "armor_bash": 2,
+    "starting_ammo": { "9mm": 30 },
+    "special_attacks": [
+      {
+        "type": "gun",
+        "cooldown": 10,
+        "move_cost": 150,
+        "gun_type": "hk_mp5",
+        "ammo_type": "9mm",
+        "fake_skills": [ [ "gun", 5 ], [ "smg", 5 ] ],
+        "fake_dex": 9,
+        "ranges": [ [ 0, 14, "DEFAULT" ] ],
+        "require_targeting_npc": true,
+        "require_targeting_monster": true,
+        "laser_lock": false,
+        "targeting_cost": 400,
+        "targeting_timeout_extend": -10,
+        "targeting_sound": "\"You should walk away.\"",
+        "targeting_volume": 20,
+        "no_ammo_sound": "a chk!"
+      }
+    ],
+    "vision_day": 30,
+    "vision_night": 10,
+    "path_settings": { "max_dist": 10 },
+    "anger_triggers": [ "HURT", "PLAYER_NEAR_BABY" ],
+    "fear_triggers": [ "SOUND" ],
+    "death_function": [ "NORMAL" ],
+    "harvest": "demihuman_large_fur",
+    "reproduction": { "baby_monster": "mon_uplifted_bear_cub", "baby_count": 1, "baby_timer": 700 },
+    "//": "Uplifts will likely outlive mankind",
```

Comment:
What did he mean by this 🤔

Reply:
When I copy pasted from bear there was a // note about how long bears pregnancy lasts.  It was late at night and I decided to keep in the reproduction and then replaced the note with a joke comment.  Now I'm thinking I might keep it as a reminder to add some reproduction to some of the further uplift monsters or to even add locations for uplifted communities of npcs.

Reply:
Huh. I always thought that the Uplifts will take over after humanity's extinction. It's a thing in some sci-fi works.